src/ui/templates/partials/quote-uk-goods-and-services-details.njk

```diff
@@ -0,0 +1,19 @@
+{% set detailsHtml %}
+  {% include "partials/uk-goods-and-services-description.njk" %}
+
+  <p data-cy="goods-services-will-calculate-thoroughly">{{ CONTENT_STRINGS.WILL_CALCULATE_THOROUGHLY }}</p>
+{% endset %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters-from-desktop">
+
+    {{ govukDetails({
+      summaryText: CONTENT_STRINGS.UK_GOODS_AND_SERVICES_DESCRIPTION.INTRO,
+      html: detailsHtml,
+      attributes: {
+        "data-cy": "goods-services"
+      }
+    }) }}
+
+  </div>
+</div>
```

Comment:
Could we have insurance and quote specific partials in their own directory? e.g `quote/uk-goods...njk` insurance/end-buyer-details.njk`